server/controllers/user.ts

```diff
@@ -134,21 +138,24 @@ export function processLogoutPage(req:Request, res: Response, next: NextFunction
 
 export function processEditPage(req: Request, res: Response, next: NextFunction): void {
     const id = req.params.id;
+    if(req.user?._id == id)
```

Comment:
The `else` part is missing here. If we don't do anything, our server will not send anything to the user request. Maybe return `401 Unauthorized` or just redirect to the account page.